contracts/Everlong.sol

```diff
@@ -365,6 +408,66 @@ contract Everlong is IEverlong {
         return output;
     }
 
+    /// @dev Calculate the losses incurred from closing sufficient positions
+    ///      to receive at least `_targetOutput`.
+    /// @param _targetOutput Target asset output from closed positions.
+    /// @return losses Losses incurred from closing the immature positions.
+    function _calcImmaturePositionLosses(
+        uint256 _targetOutput
+    ) internal view returns (uint256 losses) {
+        // Initialize variables.
+        uint256 output;
+        uint256 proceeds;
+        uint256 estimatedProceeds;
+        IEverlong.Position memory position;
+        uint256 i;
+        uint256 count = _portfolio.positionCount();
+
+        // Retrieve the `weightedSpotPrice` from the most recent checkpoint.
```

Comment:
As mentioned elsewhere, I think it would be good to chat about the methodology of how we're using this.

Reply:
From our conversation offline it seems like keeping this and adding the portfolio valuation on rebalance fix will be sufficient